spec/menu_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'menu'
+
+describe Menu do
+  let(:dish) { double :dish, details: { :name => "Pie", :price => 5 } }
```

Comment:
Very good use of mocking and doubling - as far as I can tell without playing with the code your tests are isolated from class to class